pkg/cmd/generate/cluster.go

```diff
@@ -47,7 +46,7 @@ func ensureUsers(ctx *clusterContext, objsCache objectsCache) error {
 	ctx.Printlnf("-> Ensuring Users and its RoleBindings...")
 
 	for _, user := range ctx.kubeSawAdmins.Users {
-		if ctx.specificKMemberName != "" && slices.Contains(user.Selector.SkipMembers, ctx.specificKMemberName) {
+		if user.Selector.ShouldBeSkippedForMember(ctx.specificKMemberName) {
```

Comment:
should we keep the empty name check here?

Reply:
the same, the check is part of the method